.rubocop.yml

```diff
@@ -26,7 +26,7 @@ Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
   IgnoredMethods: ['describe']
-  Max: 30
```

Comment:
- [ ] [Optional] I would suggest you not change rubocop lint settings in the future, you should try to fix the issue instead